Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ccl/configuration/storage/DownloadedCCLConfigurationStorage.kt

```diff
@@ -8,7 +8,7 @@ import timber.log.Timber
 import java.io.File
 import javax.inject.Inject
 
-class CCLConfigurationStorage @Inject constructor(
+class DownloadedCCLConfigurationStorage @Inject constructor(
```

Comment:
Renamed to `DownloadedCCLConfigurationStorage` to make it clear that only the downloaded config is stored, not the merged one.

Reply:
You are right, it should actuallly be camelCase. However we are using the non-camle case CLL everywhere. I would suggest that if we want to change it, we should open another refactoring PR. What do you think @mtwalli

Reply:
Ye, @chiljamgossow  is right, we should follow camelCase

Reply:
See https://google.github.io/styleguide/javaguide.html#s5.3-camel-case

Reply:
Ok. But let's change it for all classes in a different PR after this one is merged...